docs/replication-manager/replication-manager-for-pxc.md

```diff
@@ -0,0 +1,280 @@
+      KEY `idx_host` (`host`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+    CREATE TABLE `cluster` (
+      `cluster` varchar(31) NOT NULL,
+      `masterCandidates` varchar(255) NOT NULL,
+      `replCreds` varchar(255) NOT NULL,
+      PRIMARY KEY (`cluster`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+    CREATE TABLE `link` (
+      `clusterSlave` varchar(31) NOT NULL,
+      `clusterMaster` varchar(31) NOT NULL,
+      PRIMARY KEY (`clusterSlave`,`clusterMaster`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+    CREATE TABLE `weight` (
+     `cluster` varchar(31) NOT NULL,
+     `nodename` varchar(255) NOT NULL,
+     `weight` int NOT NULL DEFAULT 0, 
+     PRIMARY KEY (`cluster`,`nodename`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+    ```
+
+    The replication table is written to by the replication manager. You do not need to adjust the table.
+
+8. The cluster table contains the details of each cluster. The cluster name is defined in the `wsrep_cluster_name` variable value:
+
+    ```bash
+    INSERT INTO `cluster` VALUES ('DC1','10.0.4.160 10.0.4.162 10.0.4.163','master_user=\'repl\', master_password=\'replpass\'');
+    INSERT INTO `cluster` VALUES ('DC2','10.0.4.164 10.0.4.165 10.0.4.166','master_user=\'repl\', master_password=\'replpass\'');
+    INSERT INTO `cluster` VALUES ('DC3','10.0.4.167 10.0.4.168 10.0.4.169','master_user=\'repl\', master_password=\'replpass\'');
+    ```
+
+    Add the required links:
+
+    ```bash
+    INSERT INTO `link` VALUES ('DC1','DC2');
+    INSERT INTO `link` VALUES ('DC1','DC3');
+    INSERT INTO `link` VALUES ('DC2','DC1');
+    INSERT INTO `link` VALUES ('DC3','DC1');
+    ```
+
+    Add the weight:
+
+    ```bash
+    INSERT INTO `weight` VALUES('DC1','DC1-1',10); 
+    INSERT INTO `weight` VALUES('DC1','DC1-2',11); 
+    INSERT INTO `weight` VALUES('DC2','DC2-1',9);
+    INSERT INTO `weight` VALUES('DC2','DC2-2',12);
+    INSERT INTO `weight` VALUES('DC3','DC3-1',11);
+    ```
+
+    The node in the cluster with the highest value is the first to be selected.
+
+9. Provision the remote clusters and start replication. 
+
+    On one of the DC1 nodes, for example DC1-1, perform a `mysqldump` with:
+
+    ```bash
+    [root@DC1-1 ~]# mysqldump -u root -p --master-data=2 --single-transaction -R -A -E > dump.sql
+    ```  
+
+10. Compress the file if it's too large. You can use Percona XtraBackup to compress the data. Copy the backup file to one node in each remote cluster, for example to DC2-1 and DC3-1. 
+
+    To restore the dump run:
+
+    ```bash
+    [root@DC2-1 ~]# mysql -u root -p < dump.sql
+    ```
+
+    and
+
+    ```bash
+    [root@DC3-1 ~]# mysql -u root -p < dump.sql
+    ```
+
+11. Configure the replication. Set up the first replication links manually. 
+
+    For example, on DC2-1 run:
+
+    ```bash
+    mysql> change master to master_host='WAN IP of DC1-1', master_user='repl', master_password='replpass', MASTER_AUTO_POSITION = 1 for channel 'DC2-DC1';
```

Comment:
https://dev.mysql.com/doc/refman/8.0/en/change-replication-source-to.html

Reply:
Updated